style.css

```diff
@@ -0,0 +1,131 @@
+/* Edit */
+
+h1~p {
+  color: black;
+}
+
+h1 {
+  color: black;
+}
+
+/* Varför fungerar inte detta? */
+section~h1 {
+  color: red;
+}
+/*
+Selektorn section~h1 fungerar inte eftersom den
+ specifikt riktar sig mot h1-element som följer 
+ direkt efter ett section-element som en syskonselektor.
+  Detta innebär att selektorn kommer att matcha alla h1-element
+   som är syskon till ett section-element och kommer efter det,
+    vilket inte är en vanlig struktur. 
+    */
+
+body {
+  font: 16px Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  background-color: #f4f4f4;
+  color: #333;
+}
+
+header {
+  background-color: #333;
+  padding: 10px 0;
+}
+
+header nav ul {
+  display: flex;
+  justify-content: center;
+  list-style: none; /* Tar bort punktlistorna från listan i headern */
+  padding: 0;
+  margin: 0;
```

Comment:
If you want to remove all default paddings and margins from all elements you can add
`
* {
...}
`
at the top of your file under body to select everything and make bulk overrides.